src/features/content-explorer/item-list/ItemList.js

```diff
@@ -188,8 +190,30 @@ const ItemList = ({
         };
     }
 
+    const addAutoSizer = WrappedComponent => {
+        return props => {
+            return isResponsive ? (
+                <div style={{ flex: 1 }}>
```

Comment:
Would it work if we pass a `style` prop to `AutoSizer`? I'm wondering if we could avoid adding this `div` if it's not necessary.

Reply:
I tried removing the `<div>` and adding the `style` directly to `<AutoSizer />`. It seems to create a side effect of increasing the height upon screen resize:

### With the `<div>`
![Move-Copy-AutoSizer-with-div](https://user-images.githubusercontent.com/6400298/177883250-f1e8d388-1c2b-4e10-b4a2-610874a4feaf.gif)

### Without the `<div>`
![Move-Copy-Modal-AutoSizer-no-div](https://user-images.githubusercontent.com/6400298/177883259-561edcab-669f-4531-a3e7-0a2eb257c03b.gif)

The modal is normally `300px` in height, without the `<div>` after resize, it expands the height.